<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemChooser</class>
 <widget class="QDialog" name="ItemChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>530</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>321</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="filterLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Filters</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>10</y>
     <width>31</width>
     <height>551</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>30</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minLevel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxLevel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Required Level</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxReqLevel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minReqLevel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>86</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Race</string>
   </property>
  </widget>
  <widget class="QComboBox" name="raceCombo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>86</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>87</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
  </widget>
  <widget class="QComboBox" name="classCombo">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>86</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QComboBox" name="colorCombo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>112</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>115</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Slot</string>
   </property>
  </widget>
  <widget class="QComboBox" name="slotCombo">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>112</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>142</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCombo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cloth</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Leather</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Plate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="leatherCheck">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="mailCheck">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="plateCheck">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="clothCheck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>530</y>
     <width>96</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="tlabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemChooser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemChooser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>624</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>ItemChooser</receiver>
   <slot>updateClickHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>importCSVFile()</slot>
  <slot>importKey()</slot>
  <slot>updateClickHandler()</slot>
 </slots>
</ui>
